tech.json

```diff
@@ -510,72 +515,72 @@
             "h_canBombThings"
           ],
           "note": [
-            "The ability to accurately place a Morph Bomb or PB in midair following a wall jump.  Then using that bomb explosion to propel Samus forward.",
+            "The ability to accurately place a Morph Bomb or Power Bomb in mid-air following a wall jump, then using that bomb explosion to propel Samus forward.",
             "There is a timing component where the bomb is placed while rising then hit while falling, and a momentum component for maximizing horizontal distance."
           ]
         },
         {
           "name": "canBombHorizontally",
           "requires": [],
           "note": [
-            "Place a single bomb then move Samus to the side to be boosted away from the explosion.",
-            "This is used as a way to cross gaps, or to reposition at the end of an IBJ.",
+            "Place a single bomb then move Samus to the side to be boosted horizontally from the explosion.",
+            "This is typically used as a way to cross gaps or to reposition at the end of an IBJ.",
             "Press forward while in the air to carry the momentum farther.",
-            "By moving forward just before the explosion the momentum will be greater, but this is hard to do.",
-            "Since it uses one bomb, this works with a Power Bomb or a Morph Ball Bomb."
+            "Since it only uses one bomb, it can be executed with a Power Bomb or a Morph Bomb.",
+            "It is not assumed by this tech, but it is possible to move forward just before the explosion in order to gain more horizontal momentum."
           ],
           "extensionTechs": [
             {
               "name": "canHBJ",
               "requires": [ "canBombHorizontally" ],
               "note": [
-                "A Horizontal Bomb Jump (HBJ) uses two closely spaced bomb explosions to propel Samus forward farther than a single bomb can.",
-                "Placing both bombs also requires some initial momentum from either a jump morph or a third bomb explosion."
+                "A Horizontal Bomb Jump (HBJ) uses three Morph Bombs to execute. Place a bomb to get boosted horizontally; place a second bomb immediately as Samus is propelled.",
+                "Place a third bomb, mid-air, as far horizontally as possible while still being able to return and get propelled by the second and then third bombs."
```

Comment:
From a walljump it's only two bombs.  A straight jump morph can work but doesn't have any uses I don't think.  I might even be better to stop thinking about walljump HBJs and use diagonals in those cases.